Tests/GRPCHTTP2CoreTests/GRPCMessageFramerTests.swift

```diff
@@ -22,9 +22,9 @@ import XCTest
 final class GRPCMessageFramerTests: XCTestCase {
```

Comment:
Can you add some tests which check that promises are cascaded as expected?

Reply:
I added a test checking this to the `GRPCStreamStateMachine` tests, but I also just added (or rather modified an existing one) in the framer tests.